docs-devsite/auth.auth.md

```diff
@@ -144,6 +145,19 @@ const result = await signInWithEmailAndPassword(auth, email, password);
 
 ```
 
+## Auth.authStateReady()
+
+return a promise that resolves immediately when the initial auth state is settled. The current user might be a valid user, or null if there is no user signed in currently.
```

Comment:
I think this should be "Returns a promise . . ."

Along with the other fixes suggested above.